tests/modeladmin/test_checks.py

```diff
@@ -960,6 +1024,33 @@ class TestModelAdmin(ModelAdmin):
             'admin.E106'
         )
 
+    def test_invalid_model(self):
+
+        class ValidationTestInline(TabularInline):
+            model = 'Not a class'
+
+        class TestModelAdmin(ModelAdmin):
+            inlines = [ValidationTestInline, ]
```

Comment:
No need for trailing , in lists. That only applies to tuples. You can use lists throughout the patch.